src/app/Upload.svelte

```diff
@@ -3,6 +3,7 @@
 
   let onDraggedOver = false;
   let files = new Map<String, File>();
+  let downloadURL = "";
```

Comment:
```suggestion
  let downloadURL:  string;
```
**[Suggestion]**: If this string is not mutable I would use constant

Reply:
`downloadURL` is mutated below (Svelte compiles the mutation to an `$$invalidate()` call).

Reply:
Hey @IanSeng. Thanks for the review comments. I've thought about this, and I notice many of your review comments are more about style than content. In this case, you would have noticed that `downloadURL` is mutable and cannot be turned into a constant by this line:

https://github.com/eddieantonio/predictive-text-studio/pull/78/files#diff-ce0dbfe92e42d16b2e8ce7fc58fdae2fR58

What's more, we have an `eslint` rule that would fail the build if we declare an identifier as `let` when it is never assigned, so this particular comment would have already been caught by `eslint` before getting to this point :)

Please be more thorough in code review, and attempt to comment more on _content_ and _structure_ and less on _style_. If you have an issue with the style of the PR, take that up with our eslint and svelte-check rules! That's what they're there for. Let them do the talking :) Please review with concerns that only a human can check. Thanks!

Reply:
Hey @eddieantonio  I can definitely see you point and I hope that I do not come across as rude. Since we are working on a project as a team and I personally think it would be valuable to have consistent style/naming convention for the reason of maintainability and code readability. I can totally understand your perspective and will be more than happy take your feedback. Thank you.

Reply:
URL or Url, HTTP or Http, HTML or Html

Reply:
It seems our convension is the following:

```suggestion
  let downloadUrl = "";
```

Needs to be changed in the rest of the file, though!